src/globals/scss/_theme.scss

```diff
@@ -107,7 +107,7 @@
   $brand-02: $ibm-colors__blue--80 !default !global;
   $brand-03: $ibm-colors__blue--60 !default !global;
   $inverse-01: $ibm-colors__white !default !global;
-  $inverse-02: $ibm-colors__white !default !global; // TODO: Define for experimental
+  $inverse-02: #272d33 !default !global; // TODO: Define for experimental
```

Comment:
What is the reasoning for this color being changed? Is this something that should be updated in ibm-colors package? (that's where these were copied from)

Reply:
I changed it at first because it didn't match the carbon color system, wasn't sure why some variables were different colors...I'll change it back just because I don't need that color anymore this was for the "dark" duo theme but since we are only doing light theme it's not needed.